source/git-util.js

```diff
@@ -108,3 +109,14 @@ exports.verifyTagDoesNotExistOnRemote = async tagName => {
 exports.commitLogFromRevision = revision => execa.stdout('git', ['log', '--format=%s %h', `${revision}..HEAD`]);
 
 exports.push = () => execa('git', ['push', '--follow-tags']);
+
+const gitVersion = async () => {
+	const {stdout} = await execa('git', ['version']);
+	return stdout.match(/git version (\d+\.\d+\.\d+).*/)[1];
+};
+
+exports.verifyRecentGitVersion = async () => {
+	if (semver.lt(await gitVersion(), '2.11.0')) {
```

Comment:
I just noticed that this could be moved to a separate method in `version.js` named `isVersionLower`, along with `isVersionGreater`. Doing so would keep this file (almost) entirely focused on git-related methods.
Also, we're currently not checking if the output returned by `gitVersion` is actually a version, which could lead to some issues in future versions of git.
@sindresorhus Thoughts?

Reply:
> `isVersionLower`, along with `isVersionGreater`.

We don't really need both of these though. How about just one called `isVersionGreaterThanOrEqualTo()`? That's a mouthful though... Alternatively, `isMinimumVersion()` / `isAtLeastVersion()`.

> Also, we're currently not checking if the output returned by `gitVersion` is actually a version, which could lead to some issues in future versions of git.

We should definitely do that.

Reply:
> We don't really need both of these though. How about just one called `isVersionGreaterThanOrEqualTo()`? That's a mouthful though... Alternatively, `isMinimumVersion()` / `isAtLeastVersion()`.

In [`ui.js`](https://github.com/sindresorhus/np/blob/6df47b909be697687538b52f00f8b5875738933e/source/ui.js#L90) and [`prerequisite-tasks.js`](https://github.com/sindresorhus/np/blob/6df47b909be697687538b52f00f8b5875738933e/source/prerequisite-tasks.js#L53) we're using `!isVersionGreater()`, which is the same as `isVersionLowerThanOrEqualTo()`. This requires us to have a way to check if `v1 >= v2` as well as if `v1 <= v2`, which means we'll need to have at least 2 methods.

To solve this, we could keep `isVersionGreater()` and add another method for checking version equality `isVersionEqualTo()` (which is more readable in my opinion).

Reply:
Sounds good. Then we should also rename `isVersionGreater()` => `isVersionGreaterThan()` for consistency.